src/System.Diagnostics.Process/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
+      uapaot-Windows_NT;
```

Comment:
no, I haven't - this behavior seems intuitive to me and I do not believe this is an issue. cc: @weshaggard